utils/utils.jsx

```diff
@@ -160,7 +159,7 @@ export function getChannelURL(channel, teamId) {
     return notificationURL;
 }
 
-export function notifyMe(title, body, channel, teamId, silent) {
+export function notifyMe(title, body, channel, teamId, silent, state) {
```

Comment:
This can also be refactored into an action creator. That would avoid calling the global `getState` or `dispatch`.